.build/config/cassandra_ci.yaml

```diff
@@ -0,0 +1,342 @@
+  # testing. We expect these to fail even more rarely than post-commit.
+  - name: nightly
+    jobs:
+      - stress-test
+      - fqltool-test
+      - test-burn
+
+#-----------------------------------------------------------------------------
+# JOBS
+#
+# Parameters:
+#   job: the name
+#   runtime_env: which runtime_env above to base self off
+#   description: text description of the job
+#   cmd: command to run in shell to execute tests
+#   testlist: command to run in shell to generate full list of tests to run. Expected to output filename value to TESTLIST= env var.
+#   test_timeout: timeout for the test in ms
+#   multiplex: Number of times to multiplex a test of the given suite on addition or change
+#
+# Required env vars:
+#   DIST_DIR=<C*/build dir>
+#   CASSANDRA_DIR=<C* dir>
+#   TESTLIST=<populated by testlist: tag entry on a given job>
+#   JOB=<populated by job: tag entry on a given job>
+#
+# Jobs can replace any base params defined in their resource class, and map
+# types will append new or overwriting keys.
+#
+#-----------------------------------------------------------------------------
+# RUNTIME ENVIRONMENTS
+# i.e. common resource classes
+#-----------------------------------------------------------------------------
+runtime_env:
+  - type: jvm
+    resources: {cpu=4, memory=4g, storage=25g}
+    env:
+      ANT_TEST_OPTS: "-Dno-build-test=true"
+    testlist: find test/unit -name '*Test.java' | sed "s;^test/unit/;;g" | sort
+    test_timeout: 480000
+    multiplex: 500
+
+  - type: jvm-dtest
+    resources: {cpu=4, memory=16g, storage=25g}
+    env:
+      ANT_TEST_OPTS: "-Dno-build-test=true"
+    test_timeout: 900000
+    multiplex: 500
+
+  - type: python
+    resources: {cpu=4, memory=6g, storage=50g}
+    testlist: echo "WARNING! You have to define a testlist entry for this job." && exit 1
+    test_timeout: 900000
+    multiplex: 500
+    cmd: pytest ${PYTEST_OPTS} --cassandra-dir=${CASSANDRA_DIR} --keep-failed-test-dir ${DTEST_ARGS} ${TESTLIST}
+    env:
+      PYTHONIOENCODING: "utf-8"
+      PYTHONUNBUFFERED: true
+      CASS_DRIVER_NO_EXTENSIONS: true
+      CASS_DRIVER_NO_CYTHON: true
+      CCM_MAX_HEAP_SIZE: "1024M"
+      CCM_HEAP_NEWSIZE: "512M"
+      CCM_CONFIG_DIR: ${DIST_DIR}/.ccm
+      NUM_TOKENS: "16"
+      CASSANDRA_SKIP_SYNC: true
+      TMPDIR: "$(mktemp -d /tmp/run-python-dtest.XXXXXX)"
+      DTEST_ARGS: "--use-vnodes --num-tokens=${NUM_TOKENS} --skip-resource-intensive-tests"
+      # TODO: Confirm usage of --junit-prefix; see if we can just use the job name below or if these are specific (i.e. logging or control)
+      PYTEST_OPTS: "-vv --log-cli-level=DEBUG --junit-xml=${DIST_DIR}/test/output/nosetests.xml --junit-prefix=${JOB} -s"
+
+
+#-----------------------------------------------------------------------------
+# JVM based single node tests
+#-----------------------------------------------------------------------------
+test_suites:
```

Comment:
what's the diff between a `job` and a `test_suite`?  Can we just use one naming?

Reply:
Yep, for sure. Fixed.

I didn't do a _great_ job of setting expectations on this - was wanting to get some higher level "does this structure make more sense than the previous" kind of sniff testing. Appreciate the detail checking though!